pkgs/tools/security/pass/default.nix

```diff
@@ -40,8 +40,6 @@ assert x11Support -> xclip != null;
 assert waylandSupport -> wl-clipboard != null;
```

Comment:
Those two can also go

Reply:
I've removed the assertions and squashed the commits.